style/checkbox.less

```diff
@@ -8,6 +8,15 @@
   right: 0;
   bottom: 0;
   left: 0;
+
+  &.disabled {
```

Comment:
Should use `.rdg-checkbox-label-disabled`, and we can move it out of the parent selector.